app.py

```diff
@@ -19,26 +18,49 @@ def index():
     return {"message": data}
 
```

Comment:
The route decorator '@app.route' should be followed by a new line for better readability.